scripts/ci/update-open-pr

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git clone --depth 1 -b dcos-ui/latest https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos dcos-latest
```

Comment:
Shellcheck [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086) tells me that it would be nice to have double quotes around the URL here